core/src/banking_stage/latest_unprocessed_votes.rs

```diff
@@ -262,49 +262,41 @@ impl LatestUnprocessedVotes {
             &bank,
             self.latest_votes_per_pubkey.read().unwrap().keys(),
         )
         .collect_vec();
-        pubkeys_by_stake
-            .into_iter()
-            .filter(|&pubkey| {
-                if !continue_forwarding {
-                    return false;
-                }
-                if let Some(lock) = self.get_entry(pubkey) {
-                    let mut vote = lock.write().unwrap();
-                    if !vote.is_vote_taken() && !vote.is_forwarded() {
-                        let deserialized_vote_packet = vote.vote.as_ref().unwrap().clone();
-                        if let Some(sanitized_vote_transaction) = deserialized_vote_packet
-                            .build_sanitized_transaction(
-                                &bank.feature_set,
-                                bank.vote_only_bank(),
-                                bank.as_ref(),
-                            )
-                        {
-                            if forward_packet_batches_by_accounts.try_add_packet(
-                                &sanitized_vote_transaction,
-                                deserialized_vote_packet,
-                                &bank.feature_set,
-                            ) {
-                                vote.forwarded = true;
-                            } else {
-                                // To match behavior of regular transactions we stop
-                                // forwarding votes as soon as one fails
-                                continue_forwarding = false;
-                            }
-                            return true;
+        let mut count: usize = 0;
+        for pubkey in pubkeys_by_stake {
+            if let Some(lock) = self.get_entry(pubkey) {
+                let mut vote = lock.write().unwrap();
+                if !vote.is_vote_taken() && !vote.is_forwarded() {
+                    let deserialized_vote_packet = vote.vote.as_ref().unwrap().clone();
+                    if let Some(sanitized_vote_transaction) = deserialized_vote_packet
+                        .build_sanitized_transaction(
+                            &bank.feature_set,
+                            bank.vote_only_bank(),
+                            bank.as_ref(),
+                        )
+                    {
+                        if forward_packet_batches_by_accounts.try_add_packet(
+                            &sanitized_vote_transaction,
+                            deserialized_vote_packet,
+                            &bank.feature_set,
+                        ) {
+                            vote.forwarded = true;
                         } else {
-                            return false;
+                            // To match behavior of regular transactions we stop
+                            // forwarding votes as soon as one fails. We are assuming that failure (try_add_packet) means no more space available.
```

Comment:
Convention is to wrap on 100 characters for comments.

```suggestion
                            // To match behavior of regular transactions we stop forwarding votes as
                            // soon as one fails. We are  assuming that failure (try_add_packet)
                            // means no more space available.
```